exchange/utils.py

```diff
@@ -52,9 +52,17 @@ def insert_many(objects, using="default"):
     table = model._meta.db_table
     column_names = ",".join(con.ops.quote_name(f.column) for f in fields)
     placeholders = ",".join(("%s",) * len(fields))
-    con.cursor().executemany("insert into %s (%s) values (%s)"
-                             % (table, column_names, placeholders), parameters)
-    transaction.commit_unless_managed(using=using)
+
+    from django import VERSION
+    # see https://docs.djangoproject.com/en/1.9/internals/deprecation/#deprecation-removed-in-1-8
+    if VERSION[2] < 7:
```

Comment:
Hey, sorry for taking so long to see this!
I'll update this, can remove the version check if we should drop 1.6.